Examples/Algorithms/Geant4HepMC/include/ActsExamples/Geant4HepMC/EventRecording.hpp

```diff
@@ -35,6 +38,13 @@ class EventRecording final : public ActsExamples::BareAlgorithm {
     int seed1 = 12345;
     /// random number seed 2
     int seed2 = 45678;
+
+    /// List of processes that can be combined to a single vertex
+    std::vector<std::string> processesCombine;
+    /// Selective recording based on a process
```

Comment:
Thanks for updating the names, they are much more consistent now. To be a bit pedantic: from the documentation and the name alone it is still not clear to me, if an empty `processSelect` means to select nothing at all or to ignore the setting.

Reply:
I hope that it's clear now^^